### PR TITLE
Add ShapeShift FOX Token

### PR DESCRIPTION
**Pull request by willyogo**
FOX on Ethereum Mainnet: https://etherscan.io/token/0xc770eefad204b5180df6a14ee197d99d808ee52d
FOX on Optimism Mainnet, deployed by the standard token factory:  https://optimistic.etherscan.io/address/0xF1a0DA3367BC7aa04F8D94BA57B862ff37CeD174#readContract#F4

### CODE REVIEW

**Comment by jakim929** [comment]
Thank you for the PR.

@willyogo  can you update the `data.json` to have the `"name": "FOX"`, It's currently `"ShapeShift FOX"` and it's not consistent with the name on the 0xc770eefad204b5180df6a14ee197d99d808ee52d contract

Alternatively, you can use [overrides](https://github.com/ethereum-optimism/ethereum-optimism.github.io#per-token-overrides) to have different names than the contract

**Review by jakim929 — CHANGES_REQUESTED** [review]
Thank you for the PR.

@willyogo  can you update the `data.json` to have the `"name": "FOX"`, It's currently `"ShapeShift FOX"` and it's not consistent with the name on the 0xc770eefad204b5180df6a14ee197d99d808ee52d contract

Alternatively, you can use [overrides](https://github.com/ethereum-optimism/ethereum-optimism.github.io#per-token-overrides) to have different names than the contract

**Comment by willyogo** [comment]
> Thank you for the PR.
> 
> @willyogo can you update the `data.json` to have the `"name": "FOX"`, It's currently `"ShapeShift FOX"` and it's not consistent with the name on the 0xc770eefad204b5180df6a14ee197d99d808ee52d contract
> 
> Alternatively, you can use [overrides](https://github.com/ethereum-optimism/ethereum-optimism.github.io#per-token-overrides) to have different names than the contract

thanks for the quick review! just added an override; hope that takes care of it, but please lmk if not

File: data/FOX/data.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "ShapeShift FOX",
+    "symbol": "FOX",
+    "decimals": 18,
+    "description": "FOX is an ERC-20 token created by ShapeShift which serves as the governance token for the ShapeShift DAO. The token supports and provides utility for a borderless, cross-chain crypto trading platform and portfolio manager for user self-sovereignty.",
+    "website": "https://shapeshift.com",
+    "twitter": "@ShapeShift",
+    "tokens": {
+      "ethereum": {
+        "address": "0xc770eefad204b5180df6a14ee197d99d808ee52d"
```
Comment (jakim929) [inline data/FOX/data.json:10]: missing comma here

**Review by jakim929 — DISMISSED** [review]
great! one more request change here

File: data/FOX/data.json (hunk 1 above)
Comment (willyogo) [reply data/FOX/data.json]: whoops, just fixed, thank you!

File: data/FOX/data.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "symbol": "FOX",
+    "decimals": 18,
+    "description": "FOX is an ERC-20 token created by ShapeShift which serves as the governance token for the ShapeShift DAO. The token supports and provides utility for a borderless, cross-chain crypto trading platform and portfolio manager for user self-sovereignty.",
+    "website": "https://shapeshift.com",
+    "twitter": "@ShapeShift",
+    "tokens": {
+      "ethereum": {
+        "address": "0xc770eefad204b5180df6a14ee197d99d808ee52d",
+        "overrides": {
+          "name": "FOX"
```
Comment (jakim929) [inline data/FOX/data.json:12]: @willyogo  missing `},` after this line

**Review by jakim929 — APPROVED** [review]

File: data/FOX/data.json (hunk 2 above)
Comment (jakim929) [reply data/FOX/data.json:12]: looks great now! just approved.

**Pull request merged**